packages/truffle/package.json

```diff
@@ -4,7 +4,11 @@
   "version": "5.0.0-next.14",
   "description": "Truffle - Simple development framework for Ethereum",
   "dependencies": {
+    "eslint": "^5.7.0",
+    "go-ipfs-dep": "^0.4.17",
```

Comment:
What's this one for?

Reply:
It's actually the package on which the error was thrown originally: 

~~~~  error /Users/fainashalts/truffle-dev/truffle/node_modules/go-ipfs-dep: Command failed.
Exit code: 1
Command: node src/bin.js
Arguments: 
Directory: /Users/fainashalts/truffle-dev/truffle/node_modules/go-ipfs-dep
Output:
fs.js:25
'use strict';
^

ReferenceError: internalBinding is not defined
    at fs.js:25:1
    at req_ (/Users/fainashalts/truffle-dev/truffle/node_modules/natives/index.js:137:5)
    at Object.req [as require] (/Users/fainashalts/truffle-dev/truffle/node_modules/natives/index.js:54:10)
    at Object.<anonymous> (/Users/fainashalts/truffle-dev/truffle/node_modules/unzip/node_modules/graceful-fs/fs.js:1:99)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32) ~~~~

Reply:
`eslint`, `husky`, and `nyc` are already devDeps in the root `package.json`. is it redundant to include them in the `packages/truffle/package.json`?

Wondering in general if it's necessary to include the `package.json` in this commit.

Reply:
As an update, I was able to bootstrap the build successfully from this branch using node v11 w/o the added changes in `package.json`.